### DISCUSSION TITLE
Preferences > Desktop > Time Shift Buffer

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#90] [Q&A] [answered]
On my Slingbox Desktop app, I have the option to enable or disable the Time Shift Buffer.  When using the virtual remote, there is significant improvement of response time when the Buffer is off.  Is there a config that will permit this in the Slinger app?

**Comment by GerryDazoo** [comment, marked as answer]
Nope, This is an issue between your media player and the slingbox. Try to minimize the buffering in your media player. I'm using ffplayer on windows with the fololwing settings and the remote response is useable. The delay introduced by my server code is measured in micoseconds.
C:\ffmpeg\ffplay.exe -infbuf -probesize 32 -analyzeduration 0  -flags nobuffer -flags low_delay 
You "can" achieve the same result with VLC but it requires much more tweeking.
